pkg/instancetype/apply/cpu_test.go

```diff
@@ -0,0 +1,605 @@
+//nolint:dupl
+package apply_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	k8sfield "k8s.io/apimachinery/pkg/util/validation/field"
+
+	v1 "kubevirt.io/api/core/v1"
+	instancetypev1beta1 "kubevirt.io/api/instancetype/v1beta1"
```

Comment:
nit - `v1beta1` for the instancetype API I tend to just use the version in instance type specific packages.